pontoon/insights/static/js/insights_tab.js

```diff
@@ -316,61 +389,73 @@ var Pontoon = (function (my) {
                 pointHoverBorderColor: style.getPropertyValue('--white-1'),
                 order: 1,
                 spanGaps: true,
+                fill: true,
+                tension: 0.4,
               },
             ],
           },
           options: {
-            legend: {
-              display: false,
-            },
-            tooltips: {
-              mode: 'index',
-              intersect: false,
-              borderColor: style.getPropertyValue('--dark-pink'),
-              borderWidth: 1,
-              caretPadding: 5,
-              xPadding: 10,
-              yPadding: 10,
-              callbacks: {
-                label(items, chart) {
-                  const { label } = chart.datasets[items.datasetIndex];
-                  return `${label}: ${items.value} days`;
+            plugins: {
+              tooltip: {
+                mode: 'index',
+                intersect: false,
+                borderColor: style.getPropertyValue('--dark-pink'),
+                borderWidth: 1,
+                caretPadding: 5,
+                padding: {
+                  x: 10,
+                  y: 10,
+                },
+                callbacks: {
+                  labelColor: function (context) {
```

Comment:
This function is also repeated many times. I would factor it out in the `insights.js` file and then call it from all the places where it's used.